ui/src/pages/dagRuns.tsx

```
@@ -37,7 +38,21 @@ const DagRuns: Component = () => {
+                dagRun={run} 
+                onDelete={() => {
+                  getDagRunPageCount()
+                    .then((count) => {
+                      setMaxPage(count);
+                      queryClient.invalidateQueries({
+                        queryKey: ["dags-runs"],
+                      });
+                    })
+                    .catch((error) => console.error(error));
+                }}
+              />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)